docs: fix typo (#1875)
vuejs/vitepress: docs/guide/theme-badge.md

## Customize Type Color

> you can customize `background-color` of typed `<Badge />` by override css vars. The following is the default values.

Reviewer: ```suggestion
You can customize `background-color` of badges by overriding css variables. The following are the default values:
```
